test/pgw_proxy_SUITE.erl

```diff
@@ -895,10 +907,22 @@ end_per_testcase(dns_node_selection, Config) ->
     ok = meck:unload(inet_res),
     end_per_testcase(Config),
     Config;
+end_per_testcase(sx_upf_removal, Config) ->
+    ok = meck:unload(ergw_sx_node),
```

Comment:
This was not done in previous version.
For symmetry reason it is good that it is added, but might there have been a reason not to do it before?

Reply:
the missing unload was actually causing test failures and was the reason I looked at this.